UnitTests/tOption.cpp

```diff
@@ -126,7 +126,7 @@ class LeaqueChecker
     _moved = o._moved;
     return *this; 
   }
-  LeaqueChecker()  { instances++; }
+  LeaqueChecker() : _moved(false) { instances++; }
```

Comment:
Right! I think it probably worked because we run tests in debug mode only, where arbitrary initialization happend consistently (if I rember correctly)!